Windows: Set _WIN32_WINNT to 0x0600 in qwindowstheme.cpp

The structure SHSTOCKICONINFO is only available as of Windows Vista
Fixes compilation issue when using mingw-w64

diff --git a/qtbase/src/plugins/platforms/windows/qwindowstheme.cpp b/qtbase/src/plugins/platforms/windows/qwindowstheme.cpp
index 724ea00..838bf48 100644
--- a/qtbase/src/plugins/platforms/windows/qwindowstheme.cpp
+++ b/qtbase/src/plugins/platforms/windows/qwindowstheme.cpp
@@ -32,20 +32,24 @@
 ** Foundation and appearing in the file LICENSE.GPL included in the
 ** packaging of this file.  Please review the following information to
 ** ensure the GNU General Public License version 3.0 requirements will be
 ** met: http://www.gnu.org/copyleft/gpl.html.
 **
 **
 ** $QT_END_LICENSE$
 **
 ****************************************************************************/
 
+/* The structure SHSTOCKICONINFO which is used in
+ * this file is only available as of Windows Vista */
+#define _WIN32_WINNT 0x0600
+
 #include "qwindowstheme.h"
 #include "qwindowsdialoghelpers.h"
 #include "qwindowscontext.h"
 #include "qwindowsintegration.h"
 #include "qt_windows.h"
 #include "qwindowsfontdatabase.h"
 #ifdef Q_OS_WINCE
 #  include "qplatformfunctions_wince.h"
 #  include "winuser.h"

 #endif
